### PR TITLE
Add com.quixel.Bridge

### PR DESCRIPTION
**Pull request by FakeShemp**
https://quixel.com/bridge

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.quixel.Bridge
testbot, build com.quixel.Bridge

File: com.quixel.Bridge.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-documents
+modules:
+  - name: Quixel-Bridge
+    only-arches:
```
Comment (barthalion) [inline com.quixel.Bridge.yaml]: It's not needed, `flathub.json` does that.

**Comment by nedrichards** [comment]
bot, build com.quixel.Bridge 
testbot, build com.quixel.Bridge

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/com.quixel.Bridge

Thanks!

**Pull request closed without merging**